### DISCUSSION TITLE
[@types/turndown] Some elements cannot be found.

### DISCUSSION THREAD

**Discussion by hugoovf** [DefinitelyTyped/DefinitelyTyped#57934] [Issues with a @types package] [answered]
Some elements cannot be found.

![image](https://user-images.githubusercontent.com/69076992/147874381-b330de57-0b9f-4d16-9eaf-51cf5886f749.png)

[a comment by an automated account is not shown]

**Comment by sergey-zhidkov** [comment, marked as answer]
@hugovidafe you would need to modify your tsconfig.json https://stackoverflow.com/questions/42603783/missing-basic-dom-types-in-typescript-project.
